docs/section5/Rule5-31.md

```diff
@@ -0,0 +1,46 @@
+
+# Envelope - Rule 5-31  
+
+**Rule ID:** 5-31  
+**Rule Description:** Manual fenestration shading devices, such as blinds or shades, shall be modeled or not modeled the same as in the baseline building design.  
+**Rule Assertion:** B-RMR subsurface.has_manual_interior_shades=P-RMR subsurface.has_manual_interior_shades  
+**Appendix G Section:** Section G3.1-5(a)(4) Building Modeling Requirements for the Proposed design and G3.1-5(d) Building Modeling Requirements for Baseline building  
+**Appendix G Section Reference:**  None
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Checks:** None  
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. match_data_element()
+  2. get_opaque_surface_type()
+
+## Rule Logic:
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`
+
+      - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
+        - Check if surface is above-grade wall: `if get_opaque_surface_type(surface_b) == "ABOVE-GRADE WALL":`
+
```

Comment:
I don't think we need this condition. E.g., if skylights have manual shading (the subsurface is attached to a roof instead of a wall), the rule would still apply.

Reply:
Wall check and function are removed.